src/v/pandaproxy/client/CMakeLists.txt

```diff
@@ -10,6 +10,7 @@ v_cc_library(
     fetcher.cc
     logger.cc
     producer.cc
+    record_set.cc
```

Comment:
i think the word `set` in my mind is `std::set<>`-kina thing. could this be a client_batch? how is this different from a record_batch.

Reply:
Fixed in #515